changelog.md

```diff
@@ -27,7 +36,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - nil
 
 
-## [0.5.1] - 2018-01-05
+## [0.5.1] - 2019-01-05
```

Comment:
I think this should read 2019. @SimonBiggs, can you confirm?

Reply:
Yup 2019. See https://pypi.org/project/pymedphys/#history